proxy/coco/src/request/waiting_room.rs

```diff
@@ -83,6 +83,15 @@ pub struct WaitingRoom<T, D> {
     config: Config<D>,
 }
 
+impl<T, D> Default for WaitingRoom<T, D>
```

Comment:
I actually don't think we should have this because that sets the `Config`'s delta to `0` which we don't want in a production setting. So this is a big foot gun.